File: Python/close_current_tab.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.chrome.service import Service as ChromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
import time

# create Chrome Service using the WebDriver Manager
ChromeService = ChromeService(ChromeDriverManager().install())

# Create the Chrome WebDriver
chromeDriver = webdriver.Chrome(service=ChromeService)

# Open "https://ecommerce-playground.lambdatest.io/" web page on the browser
chromeDriver.get("https://ecommerce-playground.lambdatest.io/")

# Print the title of tab in focus
print(chromeDriver.title)

# Add time Delay for clear visibility and observability
time.sleep(2)

# Close the current tab in focus
chromeDriver.close()

File: JavaSeleniumCloseTabExample/src/closetab/CloseCurrentTab.java
package closetab;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class CloseCurrentTab {

	public static void main(String[] args) {
		// No need for setProperty as ChromeDriver() will auto install 
		// required browser driver for Chrome using Selenium Manager
		WebDriver driver = new ChromeDriver();
		
		// Open the "https://ecommerce-playground.lambdatest.io/" web page on the browser
		driver.get("https://ecommerce-playground.lambdatest.io/");
		
		// Print the tile of the current tab
		System.out.println(driver.getTitle());
		
		// Add time Delay for clear visibility and observability
		try {
			Thread.sleep(2);
		} catch (InterruptedException e) {
			// Catch block to handle exception
			e.printStackTrace();
		}
		
		// Close the current tab
		driver.close();

	}

}
